### ISSUE TITLE
Druid Operator pre-requisites

### ISSUE THREAD

**Issue by soumyajose0784** [druid-io/druid-operator#132]
What are the pre-requisites to install Druid operator? Is go and operator SDK required?

**Comment by himanshug** [comment]
if you can add things to your k8s cluster using kubectl, you can follow "Install the operator"  steps described in https://github.com/druid-io/druid-operator/blob/druid-operator-0.0.4/docs/getting_started.md  

after that, you can use the instructions in "Deploy a sample Druid cluster" https://github.com/druid-io/druid-operator/blob/master/docs/getting_started.md#deploy-a-sample-druid-cluster to deploy a single node sample druid cluster.

**Comment by soumyajose0784** [comment]
Thanks @himanshug   for quick response. Do we need to build the docket to create druid-operator-image ior is there one readily available?

**Comment by AdheipSingh** [comment]
Dockerhub has hosted images for each release you can use them directly @soumyajose0784

**Comment by soumyajose0784** [comment]
Thank you. @AdheipSingh

**Comment by soumyajose0784** [comment]
we would like to build the operator image at our end, Is there a link to docker file for base image golang:1.13

**Comment by AdheipSingh** [comment]
https://github.com/druid-io/druid-operator/blob/master/Dockerfile
@soumyajose0784

**Comment by soumyajose0784** [comment]
@AdheipSingh , we are trying with a centos7 base image and we would want to install golang manually and then proceed? Do you see any challanges?
As of now it fails at step 'RUN go mod download' with error 'no module specified'

**Comment by himanshug** [comment]
>  we are trying with a centos7 base image and we would want to install golang manually and then proceed? Do you see any challanges?

No, I am not aware of anything that would require a specific linux flavor.

> As of now it fails at step 'RUN go mod download' with error 'no module specified'

you can try to prefix the command with GO111MODULE=on , if that works then you probably have a relatively older version of golang where modules were not enabled by default.
